src/authentication/authentication.handler.ts

```diff
@@ -9,29 +9,42 @@ import * as jwt from 'jsonwebtoken';
 import { ApplicationError } from '../utils/errors/applicationError';
 import { readFile } from 'fs';
 
-export class AuthenticationHandler {
+const { Strategy } = require('passport-shraga');
```

Comment:
Change to `const ShragaStrategy = require('passport-shraga').Strategy;` and use ShragaStrategy where needed.